android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -967,12 +970,11 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
                                 setSendToFromValueValidationResult(errorMessage, false, true);
                             });
                 }
-            } else if (mActivityType == ActivityType.BUY
-                    && mSelectedNetwork.chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID)) {
+            } else if (mActivityType == ActivityType.BUY) {
```

Comment:
do we support buy on all networks including test nets? maybe we should exclude test chains in that place?

Reply:
or it just gives an error `Could not get buy URL` in that case and we are ok?

Reply:
Those are excluded here with [allowBuy](https://github.com/brave/brave-core/blob/0764b44bf41eb58b3ed76a05989fcaa9d728caa9/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java#L1636) so the buy option is hidden for excluded networks.

BTW, in future, with more providers, we will have to show a message "buy not supported" based on the result of token list.

Reply:
Demo for excluded networks

[ramp-excluded-networks.webm](https://user-images.githubusercontent.com/11755381/211055720-bbb0cbc5-1e9f-4514-86de-68d408e4d057.webm)

Reply:
ok, cool!